src/olympia/addons/models.py

```diff
@@ -506,6 +506,8 @@ def force_disable(self):
                  self.slug, amo.STATUS_DISABLED)
         self.update(status=amo.STATUS_DISABLED)
         self.update_version()
+        # See: https://github.com/mozilla/addons-server/issues/13194
+        self.disable_files()
 
     def force_enable(self):
```

Comment:
I guess everyone is aware that `force_disable` won't be a reversible action any longer as `force_enable` won't re-enable the files?  (Really this change should have failed an integration test somewhere)

Reply:
Yes, and that's why I added a comment with the link to the issue (it is described there)